### ISSUE TITLE
update DEBUGMODE handling

### ISSUE THREAD

**Issue by cseppan** [CEMPD/SMOKE#1]
smk_run.csh currently looks for <program>.debug in the relevant script directory if DEBUGMODE is set. With the new build process, these executables don't exist.

**Comment by bokhaeng** [comment]
Correct. DEBUGMODE is longer supported with our recent changes but we need to enable it for any further debugging.
Please add this feature if you have an idea. I am open to any suggestion.

BH

On Apr 20, 2016, at 3:14 PM, Catherine Seppanen <notifications@github.com<mailto:notifications@github.com>> wrote:


smk_run.csh currently looks for .debug in the relevant script directory if DEBUGMODE is set. With the new build process, these executables don't exist.
